src/System.Runtime.InteropServices/tests/Configurations.props

```diff
@@ -1,10 +1,9 @@
 ﻿<Project DefaultTargets="Build">
   <PropertyGroup>
     <BuildConfigurations>
-      netstandard;
+      netcoreapp;
```

Comment:
Why do you need OS Agnostic netcoreapp? There is already a netcoreapp-Unix and netcoreapp-Windows_NT

Reply:
Because that's what was there before. I didn't change any configurations. If the configuration was agnostic before, it is agnostic after my changes. If there was a mix of agnostic and non-agnostic tfms then there is the mix as well now.

Reply:
Yeah I agree, but it doesn't really make sense, it will never be used because OS Specific are always chosen above OS Agnostic configurations.

Reply:
IF there is a matching OS specific configuration...

Reply:
Ahgg I think I forgot about `FreeBSD`